apps/finance/contracts/Finance.sol

```diff
@@ -781,6 +820,18 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
         return nextPayment;
     }
 
+    // Syntax sugar
+
+    function _arr(address _a, address _b, uint256 _c, uint256 _d, uint256 _e, uint256 _f) internal pure returns (uint256[] r) {
```

Comment:
This is a bit awkward; for some reason Solidity won't let me use just the `arr()` name :(.

Don't really think we need to port this over back to aragonOS; this is a really specific helper.

Reply:
Yes, it looks like. Although that seems to me like a hard question (which ones to include in aragonOS helpers), the possible combinations of parameters are endless.